lib/cmds/push.js

```diff
@@ -136,22 +105,13 @@ async function updateMainFromFolder(
         'HEAD',
         signature,
         signature,
-        clientCommitMessage,
+        commitMsg || 'feat: Implementation [' + branchName + ']',
```

Comment:
It can't be feature  all the time
```suggestion
        commitMsg || `[${branchName}] : Implementation`,
```